src/backend/config.ts

```diff
@@ -475,8 +475,8 @@ class GlobalConfigV0 extends GlobalConfig {
       enableUpdates: false,
       addDesktopShortcuts: false,
       addStartMenuShortcuts: false,
-      autoInstallDxvk: false,
-      autoInstallVkd3d: false,
+      autoInstallDxvk: true,
+      autoInstallVkd3d: true,
```

Comment:
Just to confirm, this is going to change the default value for these options, I'm ok with this default but I'm checking if this is intended in this PR

Reply:
This should have been changed a while back, it is an old discussion. since it is 2 lines of code I will change in this PR instead of a separate one.
I will add a note on the PR description.